### PR TITLE
Wc shop choose fit

### PR DESCRIPTION
**Pull request by wcampbell91**

### CODE REVIEW

**Review by nathanieltylerowens — APPROVED** [review]
👍

File: main.js [hunk 1]
```diff
@@ -183,28 +256,12 @@ const clickEvents = () => {
   }
 }
 
-const buildAboutFrames = () => {
```
Comment (petestewart) [inline main.js:186]: Oops! My code got killed. :(

Comment (MLogan1998) [inline main.js:186]: Get Petes code back!

**Review by MLogan1998 — CHANGES_REQUESTED** [review]

**Review by petestewart — CHANGES_REQUESTED** [review]
Needs code back that got trashed

**Review by petestewart — APPROVED** [review]
🙏

**Review by petestewart — APPROVED** [review]

**Review by MLogan1998 — APPROVED** [review]

**Pull request merged**